src/css/_includes/global-header.scss

```diff
@@ -42,3 +42,31 @@
     }
   }
 }
+
+/*
+
+  Push anchor headers below fixed top bar.
+  Fixes https://github.com/getsentry/sentry-docs/issues/2685.
+
+*/
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.h1,
+.h2,
+.h3,
+.h4,
+.h5,
+.h6 {
+  &::before {
+    display: block;
+    content: " ";
+    margin-top: -($headerHeight + $spacer);
+    height: ($headerHeight + $spacer);
+    visibility: hidden;
+    pointer-events: none;
+  }
```

Comment:
How about this?

```suggestion
  scroll-margin-top: $headerHeight + $spacer;
```

Reply:
Let me check how that works on safari, there's supposed to be a problem:
https://caniuse.com/?search=scroll-margin-top

Reply:
_Sigh_, yep, Safari is not happy with that one - we can go with your solution :)